app/forms/openid_connect_logout_form.rb

```diff
@@ -4,6 +4,8 @@ class OpenidConnectLogoutForm
   include RedirectUriValidator
 
   ATTRS = %i[
+    client_id
+    current_user
```

Comment:
we can remove this since there's a private `attr_reader` for `current_user` below

Reply:
good catch!